packages/paypal-commerce-integration/src/paypal-commerce-script-loader.ts

```diff
@@ -137,14 +146,22 @@ export default class PayPalCommerceScriptLoader {
                     'payment-fields',
                     'legal',
                     ...googlePayComponent,
+                    ...enableConnectComponent,
                 ],
                 currency: currencyCode,
                 intent,
                 ...(isDeveloperModeApplicable && { 'buyer-country': buyerCountry }),
             },
             attributes: {
                 'data-partner-attribution-id': attributionId,
-                'data-client-token': clientToken,
+                ...(shouldEnableConnectComponent
```

Comment:
Should we do an update here `packages/google-pay-integration/src/google-pay-paypal-commerce/google-pay-paypal-commerce-script-loader.ts` too?

Reply:
We don't need to do it yet, because for PPCP AXO beta we will disable wallet buttons on checkout page due to some issues with configuration. Also we will create paypal commerce utils package which can be use as a single source of truth for paypal sdk configuration (no need to do extra work for now)

Reply:
Thanks for your question